test/test_torch.py

```diff
@@ -11842,6 +11890,39 @@ def test_vander_types(self, device, dtype):
         N = [None, 0, 1, 3]
         increasing = [False, True]
 
+        for x, n, inc in product(X, N, increasing):
+            numpy_dtype = torch_to_numpy_dtype_dict[dtype]
+            pt_x = torch.tensor(x, device=device, dtype=dtype)
+            np_x = np.array(x, dtype=numpy_dtype)
+
+            # Currently `CUDA` backend is not supported as
+            # at::legacy::cumprod does not support complex dtypes.
+            if self.device_type == 'cuda':
```

Comment:
I think this check should be in complex_types, not this test?

Reply:
Yup. You are right. I guess it moved around during the rebase. 

Fixed now.